changelog.txt

```diff
@@ -1,3 +1,471 @@
+  * Bump imagebuilder to v1.1.6 in upstream
+  * Add --tz flag to create, run
+  * Print errors from individual containers in pods
+  * stop podman service in e2e tests
+  * Fix `system service` panic from early hangup in events
+  * Bump github.com/opentracing/opentracing-go from 1.1.0 to 1.2.0
+  * APIv2:fix: Handle docker volume force as expected
+  * APIv2: Add docker compatible volume endpoints
+  * Bump k8s.io/api from 0.18.4 to 0.18.5
+  * test.apiv2: add testing for container initializing
+  * Bump github.com/containers/common from 0.14.3 to 0.15.1
+  * Created timestamp returned by imagelist should be in unix format
+  * APIv2 tests: usability: better test logging
+  * docs: recommend alternatives to podman inspect
+  * utils: fix parsing of cgroup with : in the name
+  * Bump k8s.io/apimachinery from 0.18.4 to 0.18.5
+  * Set TMPDIR to /var/tmp by default if not set
+  * fix: Don't override entrypoint if it's `nil`
+  * Add a note on the APIs supported by `system service`
+  * test: add tests for --user and volumes
+  * container: move volume chown after spec generation
+  * libpod: volume copyup honors namespace mappings
+  * Set console mode for windows
+  * systemd system test: run auto-update
+  * Allow empty host port in --publish flag
+  * Fix a bug with APIv2 compat network remove to log an ErrNetworkNotFound instead of nil
+  * Fixes --remote flag issues
+  * Ensure umask is set appropriately for 'system service'
+  * system tests: add pod, inspect testing
+  * specgen: fix order for setting rlimits
+  * Revert sending --remote flag to containers
+  * vendor github.com/containers/common@v0.14.3
+  * podman: add new cgroup mode split
+  * systemd generate: allow manual restart of container units in pods
+  * e2e inspect: HostConfig.SecurityOpt
+  * generate systemd: improve pod-flags filter
+  * Print port mappings in `ps` for ctrs sharing network
+  * Fix python dockerpy tests
+  * Add support for dangling filter to volumes
+  * Friendly amendment for pr 6751
+  * Set syslog for exit commands on log-level=debug
+  * Add containers.conf default file for windows and MAC Installs
+  * Docs: consistency between man / --help
+  * utils: drop default mapping when running uid!=0
+  * podman run/create: support all transports
+  * Fix inspect to display multiple label: changes
+  * podman untag: error if tag doesn't exist
+  * Set stop signal to 15 when not explicitly set
+  * libpod: specify mappings to the storage
+  * APIv2: Return `StatusCreated` from volume creation
+  * APIv2:fix: Remove `/json` from compat network EPs
+  * Fix ssh-agent support
+  * APIv2:doc: Fix swagger doc to refer to volumes
+  * BATS tests: new too-many-arguments test
+  * Reformat inspect network settings
+  * Add podman network to bash command completions
+  * Fix typo in manpage for `podman auto update`.
+  * Add tests for --privileged with other flags
+  * Add JSON output field for ps
+  * V2 podman system connection
+  * wip
```

Comment:
Did we really merge a PR only with "wip" as a title?

Reply:
Yes :-D